### ISSUE TITLE
Bad link in twisted web documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7125]
|<img alt="Tekhne's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Tekhne reported|
|-|-|
|Trac ID|trac#1404|
|Type|defect|
|Created|2006-01-06 06:29:21Z|

Attachments:

* [bad_link_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b9b/b9b31b56d8b82c95acf0f300853d587c81b0ebc0/ce7408e36d28f9a7fa6daaf0255e6a54211aaf16.patch) (706 bytes) - added by khorn on 2009-10-29 18:32:03Z - fix broken link

<details><summary>Searchable metadata</summary>

```
trac-id__1404 1404
type__defect defect
reporter__Tekhne Tekhne
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1136528961000000 1136528961000000
changetime__1474197555542286 1474197555542286
version__ 
owner__ 
cc__Tekhne cc__thijs cc__exarkun cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Tekhne's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Tekhne commented|
|-|-|

```
#!html
<pre>
FYI: 

http://twistedmatrix.com/projects/web/documentation/howto/xmlrpc.html 

links to:

http://twistedmatrix.com/projects/web/documentation/howto/defer.html

...but the latter doesn't seem to exist.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement set owner to edsuom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @radix|
|-|-|

Boy, this sure is terrible, two years later.

In a subproject howto, there's no way to express a link to a howto in a different subproject.  This is a documentation generation issue, not just a web defect.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

also see [#3177](https://github.com/twisted/twisted/issues/3177)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Blocked by [#3438](https://github.com/twisted/twisted/issues/3438) then

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

Restoring the original ticket summary, since [#3438](https://github.com/twisted/twisted/issues/3438) seems to cover the general cross-subproject linking issue.  Let's just make this about the particular link in the Twisted Web docs which is broken.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn set owner to @khorn|
|-|-|

patch to fix broken link

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27431]) Apply bad_link_fix.patch: fix broken Deferred link in xmlrpc documentation.

Author: khorn
Reviewer: thijs
Fixes: [#7125](https://github.com/twisted/twisted/issues/7125)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_11"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
